docs/protocol/src/protocol/addresses_keys/spend_key.md

```diff
@@ -9,8 +9,10 @@ PBKDF2 with:
 * `mnemonic` concatenated with an index used as the salt, i.e. the
 default spend authority is derived using the salt `mnemonic0`
 
+TODO: describe new BIP44 derivation and change raw BIP39 derivation to be described as "legacy".
+
 The root key material for a particular spend authority is the 32-byte
-`seed` derived as above from the seed phrase. The `seed` value is used to derive
+`spend_key_bytes` derived as above from the seed phrase. The `spend_key_bytes` value is used to derive
```

Comment:
I changed this to eliminate references to `seed`; I think it's confusing since although it's the "seed" of the Penumbra-specific part of the key heirarchy, it's not necessarily the seed in the sense of the seed phrase (i.e., the ultimate root of capability).